.github/workflows/publish-helm-chart.yml

```diff
@@ -0,0 +1,42 @@
+name: Publish helm chart
+
+concurrency: release-helm
+
+on:
+  workflow_dispatch:
+  push:
+    branches:
+      - 'main'
+
+jobs:
+  release:
+    # depending on default permission settings for your org (contents being read-only or read-write for workloads), you will have to add permissions
+    # see: https://docs.github.com/en/actions/security-guides/automatic-token-authentication#modifying-the-permissions-for-the-github_token
```

Comment:
```suggestion
```

This is the permissions needed to modify a branch, so I figure the note isn't relevant to retain.